src/main/java/org/terracotta/statistics/archive/Sink.java

```diff
@@ -19,9 +19,9 @@
  * @author cdennis
  */
 @FunctionalInterface
-public interface SampleSink<T> {
+public interface Sink<T> {
 
-  static <T> SampleSink<T> devNull() {
+  static <T> Sink<T> devNull() {
```

Comment:
If we're going Java 8 then this is just a `Consumer`.

Reply:
I was wondering if we were going to keep it or not. I wanted to remove it also.

Reply:
You have my blessing if you want to remove it with this PR.